### ISSUE TITLE
Cannot read property 'status' of undefined

### ISSUE THREAD

**Issue by numediaweb** [mysiar/api-platform-doc-parsing-check#2]
When I test it locally with my https dev domain I get:
```
node_modules/@api-platform/api-platform-doc-parsing-check/lib/index.js http://myapp-mic.de

TypeError: Cannot read property 'status' of undefined
    at /Users/aidrissi/apps/coupons/coupon-admin/node_modules/@api-platform/api-doc-parser/lib/hydra/parseHydraDocumentation.js:295:134
```

**Comment by numediaweb** [comment]
One year later... Strange that I'am the only one facing this issue :(

**Comment by alanpoulain** [comment]
Should be solved with 0.2.0, never too late!
